share/spack/gitlab/cloud_pipelines/stacks/developer-tools-darwin/spack.yaml

```diff
@@ -69,7 +69,7 @@ spack:
     - build-job:
         variables:
           CI_GPG_KEY_ROOT: /etc/protected-runner
-        tags: [ "macos-ventura", "apple-clang-15", "aarch64-macos" ]
```

Comment:
while we are at it, can these tags just go into the `configs/darwin/aarch64/ci.yaml`?

Reply:
Yes, just pushed a commit. Will you look at it and tell me if this is right?